server_src/lib/providers/settings.constants.js

```diff
@@ -0,0 +1,53 @@
+function getDefaultLayout() {
```

Comment:
Is it worth keeping these inside of lib? I personal think they would be better all under the services folder but let me know your thoughts.

Reply:
These are app constants. I am collecting them to distribute them into V2 modules later, so they would end up in a `blablab.config.ts` or `blablabla.constants.ts` in the root or config folder of a module. If the constant is really important, it would go to `app.constants.ts`.

So this is me preparing for that.

Reply:
Good to know! Cheers.